src/main/java/com/mongodb/spark/sql/connector/read/partitioner/FieldPartitioner.java

```diff
@@ -29,47 +28,45 @@
 import org.jetbrains.annotations.ApiStatus;
 
 import org.bson.BsonDocument;
-import org.bson.BsonNull;
 
 import com.mongodb.spark.sql.connector.config.ReadConfig;
 import com.mongodb.spark.sql.connector.exceptions.ConfigException;
 import com.mongodb.spark.sql.connector.read.MongoInputPartition;
 
 /**
- * Partitions collections using one or more fields.
+ * Partitions collections using a single field.
  *
  * <ul>
- *   <li>{@value PARTITION_FIELD_LIST_CONFIG}: A comma delimited list of fields to be used for
- *       partitioning. Defaults to: {@value ID_FIELD}.
+ *   <li>{@value PARTITION_FIELD_CONFIG}: A comma delimited list of fields to be used for
```

Comment:
"A comma delimited list of fields" - not a list anymore. The same is applicable to `PaginateIntoPartitionsPartitioner` and `PaginatePartitioner`.

Reply:
Good catch, Thank you